src/content/docs/cypher/import_export_db.md

```diff
@@ -0,0 +1,32 @@
+---
+title: EXPORT/IMPORT DATABASE
+---
+
+Kùzu allows you to import/export database using query statements. 
+
+## EXPORT DATABASE
```

Comment:
Headers should be title case (Only first letter is capitalized) in this case, `## Export database`